crypto/chacha/asm/chacha-x86_64.pl

```diff
@@ -440,13 +442,14 @@ sub SSSE3ROUND {	# critical path is 20 "SIMD ticks" per round
 	&por	($b,$t);
 }
 
-my $xframe = $win64 ? 32+32+8 : 24;
+my $xframe = $win64 ? 32+8 : 8;
```

Comment:
Quick question: What's the 8 for? If I'm reading this code right, it looks like the stack layout is:

```
(high address)

  [ ... previous stack frames ... ]
  [ return address ]
%r9 -->
  [ 8 bytes ]
  [ 32 bytes to save %xmm[67]; Win64-only ]
  [ 64 bytes of scratch space ]
%rsp -->

(low address)
```
where the Win64 save space is the `-0x28(%r9)` and `-0x18(%r9)` accesses in line 466 and the scratch space is the `0x00(%rsp)`, `0x10(%rsp)`, `0x20(%rsp)`, `0x30(%rsp)` accesses in the function body. Is that correct?

I'm not seeing anything which accesses the 8 bytes in front, though I may just be missing something. What's it for?

(Ditto for the other functions in this file.)